Tests/UnitTests/Mocks/MockOfferingsAPI.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  MockOfferingsAPI.swift
+//
+//  Created by Joshua Liebowitz on 6/17/22.
+
+import Foundation
+@testable import RevenueCat
+
+// swiftlint:disable line_length
+// swiftlint:disable large_tuple
```

Comment:
This can be just one line:
```suggestion
// swiftlint:disable line_length large_tuple
```

Reply:
✅